src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,61 @@
-export const Tabs = () => {};
+/* eslint-disable react/prefer-stateless-function */
+import React from 'react';
+import classNames from 'classnames';
+import { Tab } from '../../types/Tab';
+
+type Props = {
+  tabs: Tab[],
+  selectedTabId: string,
+  onTabSelected: (tab: Tab) => void,
+};
+
+export class Tabs extends React.Component<Props, {}> {
+  render() {
+    const { tabs, selectedTabId, onTabSelected } = this.props;
+
+    let selectedTab = tabs.find(
+      (tab: Tab) => tab.id === selectedTabId,
+    );
```

Comment:
```suggestion
    let selectedTab = tabs.find(
      (tab: Tab) => tab.id === selectedTabId,
    ) || tabs[0];
```

instead of `if` block below